core/rend/dx11/dx11_overlay.h

```diff
@@ -61,9 +61,14 @@ class DX11Overlay
 	ComPtr<ID3D11ShaderResourceView> xhairTextureView;
 	std::array<ComPtr<ID3D11Texture2D>, 8> vmuTextures;
 	std::array<ComPtr<ID3D11ShaderResourceView>, 8> vmuTextureViews;
-	std::array<u64, 8> vmuLastChanged {};
+	std::array<u64, 8> vmuLastChanged{};
 	Quad quad;
-	Samplers *samplers;
+	Samplers* samplers;
 	BlendStates blendStates;
 	DepthStencilStates depthStencilStates;
+	bool isDragging = false;
+	int draggedVMU = -1;
+	float dragOffsetX[8] = { 0 };
+	float dragOffsetY[8] = { 0 };
+	POINT lastMousePos = { 0, 0 };
```

Comment:
Same remark regarding VMU dragging. Other changes are white spaces. So this file doesn't need any change.